client/src/app/core/menu/menu.service.ts

```diff
@@ -7,6 +7,7 @@ import { debounceTime } from 'rxjs/operators'
 export class MenuService {
   isMenuDisplayed = true
   isMenuChangedByUser = false
+  menuWidth = 240
```

Comment:
```suggestion
  menuWidth = 240  // should be kept equal to $menu-width
```

Reply:
This allows backtracking to the source value when looking for its name. If you say there is no clean way to do it in css, then we can only do that much :/

Reply:
Thanks !